homeassistant/components/n26/__init__.py

```diff
@@ -26,6 +32,8 @@
                 {
                     vol.Required(CONF_USERNAME): cv.string,
                     vol.Required(CONF_PASSWORD): cv.string,
+                    vol.Required(CONF_ACCESS_TOKEN): cv.string,
```

Comment:
We no longer allow for changes to the YAML configuration of integrations that communicate with a device or service.

See: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

To get started on this, this integration needs to be converted to a configuration flow first. See: <https://developers.home-assistant.io/docs/config_entries_config_flow_handler>